hadoop-hdds/container-service/src/test/java/org/apache/hadoop/ozone/TestHddsDatanodeService.java

```diff
@@ -33,24 +47,48 @@
 import static org.apache.hadoop.hdds.HddsConfigKeys.HDDS_CONTAINER_TOKEN_ENABLED;
 import static org.apache.hadoop.ozone.OzoneConfigKeys.OZONE_SECURITY_ENABLED_KEY;
 import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
 import static org.junit.jupiter.api.Assertions.assertNotNull;
 import static org.junit.jupiter.api.Assertions.assertNull;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 /**
  * Test class for {@link HddsDatanodeService}.
  */
+
 public class TestHddsDatanodeService {
+
   private File testDir;
-  private OzoneConfiguration conf;
-  private HddsDatanodeService service;
-  private String[] args = new String[] {};
+  private static final Logger LOG =
+      LoggerFactory.getLogger(TestHddsDatanodeService.class);
+
+  private final String clusterId = UUID.randomUUID().toString();
+  private final OzoneConfiguration conf = new OzoneConfiguration();
+  private final HddsDatanodeService service =
+      HddsDatanodeService.createHddsDatanodeService(new String[] {});
+  private static final int SCM_SERVER_COUNT = 1;
+  private static final String FILE_SEPARATOR = File.separator;
```

Comment:
Would be better to use Ozone separator delimiter: `OzoneConts.OZONE_URI_DELIMITER `